README.md

```diff
@@ -65,15 +63,33 @@ the same syntax as `load`.  This command:
 
 ```julia
 julia> img = AstroImage("file.fits")
-AstroImages.AstroImage{UInt16,ColorTypes.Gray}[...]
+AstroImages.AstroImage{UInt16,ColorTypes.Gray,1,Float64}[...]
 ```
 
-will read the first extension from the `file.fits` file and wrap its content in
-a `Matrix{Gray}`, that can be easily used with `Images.jl` and related packages.
+will read the first valid extension from the `file.fits` file and wrap its content in
+a `NTuple{N, Matrix{Gray}}`, that can be easily used with `Images.jl` and related packages.
 
 If you are working in a Jupyter notebook, an `AstroImage` object is
 automatically rendered as a PNG image.
 
+`AstroImage` automatically extracts and store `wcs` info. of images in a `NTuple{N, WCSTransform}`.
+
+## Forming RGB image
+Using `Reproject.jl`, `AstroImage` can automatically construct a RGB image if 3 different colour band data is given.
```

Comment:
Someone reading this probably has no clue of what `Reproject.jl` is.  I'm not sure it's worth mentioning it at all here.